index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    
+    <header class="header">
+        <nav>
+          <a class="logo">My Logo</a> 
+          <button class="hamburger"><img src="./img/Enabled.png" alt=""></button>
+        </nav>
+  </header>
+  
+        <section class="main">
+            <h1 class="primary-text">I’m Nshanji. <br>
+                Glad to see you!</h1>
+            <p class="content">I’m a software developer! I can help you build a product ,
+                    feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <h2 class="social">LET'S CONNECT</h2>    
+            <div class="social-media-icons">
+                <ul>
+                    <li> <a href="#"><img src="./img/twitter-icon.svg" alt=""></a></li>
+                    <li><a href="#"><img src="./img/Linkedin-icon.svg" alt=""></a></li>
+                    <li><a href="#"><img src="./img/Matador-icon.svg" alt=""></a></li>
+                    <li><a href="#"><img src="./img/github_icon.svg" alt=""></a></li>
+                    <li><a href="#"><img src="./img/victory-icon.svg" alt=""></a></li>
```

Comment:
- [ ] **[OPTIONAL]:** You did a great job following the previous reviewer's advice! 💯 However, It would be great if you include the links to your social media, as it will help people viewing your portfolio connect easier with you! 🤓